src/test/java/com/codedifferently/collections/listTest/LinkedListTest.java

```diff
@@ -0,0 +1,63 @@
+package com.codedifferently.collections.listTest;
+
+import com.codedifferently.collections.list.ArrayList;
+import com.codedifferently.collections.list.EmptyLinkedListException;
+import com.codedifferently.collections.list.LinkedList;
+import org.junit.Assert;
+import org.junit.Test;
+
+
+
+public class LinkedListTest {
+
+    @Test
+    public void containsTest(){
+        LinkedList linked = new LinkedList();
+        linked.addHead("Paul");
+        Assert.assertTrue(linked.contains("Paul"));
+
+    }
+
+    @Test
+    public void printNodeContentTest(){
+        LinkedList linked = new LinkedList();
+        linked.addHead("Paul");
+        String expected = "Paul";
+        String actual = linked.printNodeContent();
+        Assert.assertEquals(expected, actual);
+
+    }
+
+    @Test
+    public void addHeadTest(){
+        LinkedList linked = new LinkedList();
+        linked.addHead("Paul");
+        Assert.assertTrue(linked.contains("Paul"));
+    }
+
+    @Test
+    public void addTailTest() throws EmptyLinkedListException {
+        LinkedList linked = new LinkedList();
+        linked.addHead("Paul");
+        linked.addTail("Amy");
+        Assert.assertTrue(linked.contains("Amy"));
+
+    }
+
+    @Test
+    public void DeleteHeadTest() throws EmptyLinkedListException {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.